server/graphql/v2/object/Host.js

```diff
@@ -626,15 +626,13 @@ export const Host = new GraphQLObjectType({
             const dateTo = args.dateTo ? moment(args.dateTo) : null;
             const timeUnit = args.timeUnit || getTimeUnit(numberOfDays);
 
-            const amountDataPoints = await queries.getTransactionsTimeSeries(
-              TransactionKind.EXPENSE,
-              TransactionTypes.DEBIT,
-              host.id,
-              timeUnit,
+            const amountDataPoints = await HostMetricsLib.getTransactionsTimeSeries(host.id, timeUnit, {
+              type: TransactionTypes.DEBIT,
+              kind: TransactionKind.EXPENSE,
               collectiveIds,
               dateFrom,
               dateTo,
-            );
```

Comment:
Switched this to use a query from `host-metrics.js`, where I added a new version to get the time series not grouped by `kind`, that this deprecated query did not do.

Reply:
Did this to clear up the namespace in `lib/queries.js` to add the new `getTransactionsTimeSeries` query not relying on hostId